tdrs-frontend/src/components/ResourceCards/ResourceCards.jsx

```diff
@@ -0,0 +1,128 @@
+import Button from '../Button'
+
+function ResourceCards() {
```

Comment:
coul `ResourceCards` potentially be made into a more generic component? here, i'm thinking you might have two components: `ResourceCards` and `ResourceCard`

```
const ResourceCards = (props) => (
    <section className="padding-top-4 usa-section">
        <div className="grid-row">
            {/* ... etc. */}

            {props.cards.map(card => <ResourceCard title={card.title} description={card.description}>)}
        </div>
    </section>
)
```

Hope that makes sense. I think this makes the code more declarative for the top-level implementation and provides some reusable components we can drop in other places if needed.

Reply:
@reitermb @ttran-hub -- since these resource cards are now included on more than the splash page, should these be included in the sitemap? i noticed it isn't an AC in this ticket.

Reply:
The resource cards are just a shared component, not a separate page so I don' think they need to be on the site map.